### ISSUE TITLE
Not able to run argocd commands in Windows CLI- cmd and powershell

### ISSUE THREAD

**Issue by balajirajmohan** [argoproj/argo-cd#9702]
I am trying to connect to my ArgoCD setup in my windows laptop. I am getting the below error.

PS C:\kube> ./argocd proj list
time="2022-06-17T15:35:09+05:30" level=fatal msg="config file has incorrect permission flags:-rw-rw-rw-.change the file permission either to 0400 or 0600."
PS C:\kube>

<img width="807" alt="argogitissue" src="https://user-images.githubusercontent.com/46148583/174278398-d2a21c1b-b500-4a6f-ba3c-03235af137ca.PNG">

**Comment by balajirajmohan** [comment]
Checklist:

 I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
 I've included steps to reproduce the bug.
 I've pasted the output of argocd version.

Describe the bug
I am trying to connect to my ArgoCD setup in my windows laptop. I am getting the below error.

PS C:\kube> ./argocd proj list
time="2022-06-17T15:35:09+05:30" level=fatal msg="config file has incorrect permission flags:-rw-rw-rw-.change the file permission either to 0400 or 0600."
PS C:\kube>

<img width="807" alt="argogitissue" src="https://user-images.githubusercontent.com/46148583/174278729-38dd8dfc-69c6-4d37-b950-ac682092ab66.PNG">
